### ISSUE TITLE
[GOAL] PoC implementations

### ISSUE THREAD

**Issue by antonionardella** [iota-community/X-Team_IOTA_SmartContracts#5]
# Goal Title
**Priority: (B)**
Build proof of concepts on ISCP

- ERCxx, with APIs
- CryptoKitties
- Uniswap
- Various level of complexity DeX for colored tokens
- Micropay 

# Team
IOTA ISCP - IOTA Experience Team

### X-Team members
@githubusername - Name Surname - thisis@myema.il 

### IOTA Foundation member
@githubusername - Name Surname - thisis@iota.org

# Additional Information
* link to project
* link to documentation
* link to repository
* link

# Governance
Please use the following emoji reactions in this post to indicate:
:+1: - Participating
:heavy_plus_sign: - Add my vote

**Comment by johanneskeim** [comment]
Good idea for a PoC would be an open source and non-custodial pooled liquidity protocol to earn interest on deposits and borrowed assets. 
Example: https://www.liqwid.finance/
